snap/hooks/connect-plug-peers

```diff
@@ -3,6 +3,6 @@ peer="$(snapctl get --slot :peers content)"
 (
     echo "hook $0 $@"
     echo "connected to peer $peer"
-    echo "$0 is starting $SNAP_NAME.fetch-oci"
-    snapctl start $SNAP_NAME.fetch-oci
+    echo "$0 is starting $SNAP_INSTANCE_NAME.fetch-oci"
```

Comment:
I don't know if we'd want to do something like:
`SNAP_INSTANCE_NAME=${SNAP_INSTANCE_NAME:-$SNAP_NAME}`

But otherwise this script looks fit for purpose.